htdocs/core/modules/DolibarrModules.class.php

```diff
@@ -1049,7 +1049,7 @@ function _load_tables($reldir)
 					{
 						if (preg_match('/\.sql$/i',$file) && ! preg_match('/\.key\.sql$/i',$file) && substr($file,0,4) == 'llx_' && substr($file,0,4) != 'data')
 						{
-							$result=run_sql($dir.$file,1,'',1);
+							$result=run_sql($dir.$file, $conf->global->INSTALL_DISPLAY_SQL_LOG, '', 1);
```

Comment:
Replace 
$conf->global->INSTALL_DISPLAY_SQL_LOG
with
empty($conf->global->INSTALL_DISPLAY_SQL_LOG)?0:1

to avoid warning in strict mode